test/nbrowser/DropdownConditionEditor.ts

```diff
@@ -30,8 +30,13 @@ describe('DropdownConditionEditor', function () {
           're\nc\n.Supervisor\n ',
         ]);
       });
-      await gu.sendKeysSlowly(['hoice not in $']);
+      await gu.sendKeysSlowly(['hoice not in ']);
+      // Attempts to reduce test flakiness by delaying input of $. Not guaranteed to do anything.
+      await driver.sleep(100);
+      await gu.sendKeys('$');
       await gu.waitToPass(async () => {
+        // This test is sometimes flaky here. It will consistently return the wrong value, usually an array of
+        // empty strings. The running theory is it's an issue in Ace editor.
```

Comment:
Since this is about completions, those make a request to the server, so it's expected that it takes a little time for them to show up.

Reply:
Commenting here for posterity (discussed elsewhere): it looks like the dropdown condition formula uses frontend-only autocomplete and doesn't access the server or data engine.